.github/workflows/lighthouse-ci.yml

```
@@ -29,7 +29,10 @@ jobs:
       - name: Build Next.js app
         run: npm run build
         env:
+<<<<<<< HEAD
```

actionlint (1.7.10)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)